packages/test-sync/src/androidTest/kotlin/io/realm/kotlin/test/mongodb/shared/SyncConfigTests.kt

```diff
@@ -73,6 +72,10 @@ import kotlin.time.Duration.Companion.seconds
 
 const val DEFAULT_NAME = "test.realm"
 
+internal enum class ValueType {
+    STRING, LONG, INT, NULL, OBJECT_ID, UUID
```

Comment:
Are these being used for anything?

Reply:
Yes, they are using in some tests below to do exhaustive testing see: https://github.com/realm/realm-kotlin/blob/090b4bc6bfd6ecec3955c90f6201836f962589bf/packages/test-sync/src/androidTest/kotlin/io/realm/kotlin/test/mongodb/shared/SyncConfigTests.kt#L331